synapse/lib/storm.py

```diff
@@ -390,11 +396,50 @@ def loadRuntVars(self, query):
                 self.tick()
                 yield node, path
 
-    async def getScopeRuntime(self, query, opts=None):
+    async def getScopeRuntime(self, query, opts=None, impd=False, modl=False):
         runt = Runtime(self.snap, user=self.user, opts=opts)
+        runt.isModuleRunt = modl
+        runt.isImport = impd
+        if not impd:  # respect the import boundary
```

Comment:
is this where runt.isImport should be used?

Reply:
At one point yes, and also down in propBackGlobals, but that also turned out to be unneeded. I will clean up now.